lightly/utils/optim.py

```diff
@@ -0,0 +1,90 @@
+from typing import Any, Dict, Iterable, Optional
+
+from torch.optim import Optimizer
+
+
+def update_param_groups(
+    optimizer: Optimizer,
+    default_update: Optional[Dict[str, Any]] = None,
+    updates: Optional[Iterable[Dict[str, Any]]] = None,
+) -> None:
+    """Update the values in the param groups of the optimizer.
+
+    This is useful to update values following a schedule during training.
```

Comment:
We should mention here which update is applied first so it's clear which one overrides the other.

Reply:
Added a note that `updates` has precedence over `default_updates`.